resources/js/components/ThemeOverview.vue

```diff
@@ -0,0 +1,193 @@
+<script>
+import Swiper from 'swiper';
+import noticeMixin from '../mixins/NoticeMixin';
+
+const axios = require('axios');
+
+export default {
+    name: 'ThemeOverview',
+    mixins: [noticeMixin],
+    data: () => ({
+        activeFilter: false,
+        loading: false,
+        firstLoading: true,
+        themeListNew: [],
+        themeListProgress: [],
+        themeListFinished: [],
+        themeListNewCount: 0,
+        themeListProgressCount: 0,
+        themeListFinishedCount: 0,
+        themeFirstBlock: '.ris-card-list__themes',
+        postcodeList: [],
+        selectedDistrictList: [],
+        districtList: [],
+        districtInfoDescription: 'There are no topics',
+        postcodeInfoDescription: 'There are no topics for the selected postcode',
+    }),
+    props: {
+        defaultDistrictList: {
+            type: Array,
+            default: () => [],
+        },
+    },
+    methods: {
+        collapseFilter() {
+            this.activeFilter = !this.activeFilter;
+        },
+        getTopicByDistrict(e) {
+            if (!this.loading) {
+                this.loading = true;
+                this.postcodeList = [];
+
+                this.themeListNew = [];
+                this.themeListProgress = [];
+                this.themeListFinished = [];
+                this.themeListNewCount = 0;
+                this.themeListProgressCount = 0;
+                this.themeListFinishedCount = 0;
+
+                // collapse filter block when the district is selected
+                this.collapseFilter();
+
+                const currentSelectedDistrict = e.currentTarget.innerText.trim();
+                this.selectedDistrictList.push(currentSelectedDistrict);
+
+                this.removeDistrictFromList(currentSelectedDistrict);
+
+                this.selectedDistrictList.forEach(currentDistrictName => {
+
+                    axios
+                        .get(`/api/topics?district=${currentDistrictName}`)
+                        .then(res => {
+                            if (res.data.data.length === 0) {
+                                this.info(currentDistrictName, this.districtInfoDescription);
+                            } else {
+                                res.data.data.forEach((topic) => {
+                                    if (topic.newTopic) {
+                                        this.themeListNewCount++;
```

Comment:
i think `this.themeListNewCount` and other similar in this file could be removed and `this.themeListNew.length` and others could be used instead?